### PR TITLE
Fix s3host and only report to slack for non-test

### PR DESCRIPTION
**Pull request by gabriel**
There wasn't any bug other than it reporting wrong location to slack.

### CODE REVIEW

**Comment by gabriel** [comment]
@oconnor663

File: packaging/prerelease/build_app.sh [hunk 1]
```diff
@@ -47,6 +41,14 @@ if [ ! "$bucket_name" = "" ]; then
   echo "Bucket name: $bucket_name"
 fi
 
+s3host=""
```
Comment (oconnor663) [inline packaging/prerelease/build_app.sh]: I don't think we need this line.

File: packaging/prerelease/build_app.sh [hunk 2]
```diff
@@ -111,4 +113,6 @@ BUCKET_NAME="$bucket_name" PLATFORM="$platform" "$dir/s3_index.sh"
 
 "$client_dir/packaging/slack/send.sh" "Finished $platform $build_desc (keybase: $version, kbfs: $kbfs_version). See $s3host"
 
-BUCKET_NAME="$bucket_name" "$dir/report.sh"
+if [ "$istest" = "" ]; then
```
Comment (oconnor663) [inline packaging/prerelease/build_app.sh]: I think the standard way to do this is `[ -z "$istest" ]`, but on the other hand your way doesn't require memorizing what `-z` means...I'm fine with either.

**Comment by oconnor663** [comment]
:ship:

**Pull request merged**